File: ScheduleBackendDotNet/Models/Booking.cs
using System;
using System.Collections.Generic;

namespace ScheduleBackendDotNet.Models
{
    public partial class Booking
    {
        public int BookingId { get; set; }
        public int? UserId { get; set; }
        public int? ShowId { get; set; }
        public decimal? Amount { get; set; }
        public sbyte? Status { get; set; }
        public int? SeatNumber { get; set; }

        public virtual Show? Show { get; set; }
        public virtual User? User { get; set; }
    }
}

File: moviebackend/src/main/java/com/example/demo/entities/Booking.java
package com.example.demo.entities;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@Entity
@Table(name="booking")
public class Booking {
    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    int booking_id;
    @ManyToOne
    @JoinColumn(name="user_id")

    Customer customer;
    @Column
    int show_id;
    @Column
    int amount;

    @Column
    int status;

    @Column
    int seat_number;

    public Booking(Customer customer, int show_id, int amount, int status, int seat_number) {
        this.customer = customer;
        this.show_id = show_id;
        this.amount = amount;
        this.status = status;
        this.seat_number = seat_number;
    }
}
